Misc/NEWS.d/next/Core and Builtins/2024-06-12-13-47-25.gh-issue-120397.n-I_cc.rst

```diff
@@ -0,0 +1,3 @@
+Significantly improve the speed of the str.count, bytes.count and
+bytearray.count method when the argument is a single character and the
+target architecture and compiler support vectorization.
```

Comment:
```suggestion
Improve the :meth:`str.count`, :meth:`bytes.count` and :meth:`bytearray.count`
methods for counting single characters.
```

I think we can keep it simple. You can add a WhatsNew entry with more information and indicate that this only for the case when *maxcount* is not specified.

Reply:
Maxcount is not a parameter for count. It is only used by internal function calls. So your suggestion works perfectly.